### PR TITLE
2118759: Leave manifest consumer identity certificate

### PR DESCRIPTION
**Pull request by wottop**
Plus a bit of checkstyle cleanup

### CODE REVIEW

**Review by nikosmoum — APPROVED** [review]

File: src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java [hunk 1]
```diff
@@ -75,9 +75,9 @@ protected void verifyOutput(GuestId source, GuestIdDTOArrayElement dto, boolean
             assertEquals(source.getId(), dto.getId());
             assertEquals(source.getGuestId(), dto.getGuestId());
             assertEquals(source.getCreated(), dto.getCreated() != null ?
-                new Date(dto.getCreated().toInstant().toEpochMilli()) : null);
+                (new Date(dto.getCreated().toInstant().toEpochMilli())) : null);
```
Comment (Ceiu) [inline src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java:78]: We can just use .getTime() to get the milliseconds straight from the date object

**Review by Ceiu — CHANGES_REQUESTED** [review]
We should also patch up the place where the NPE occurs so that it either completes when the cert is null, or throws an exception that is more indicative of the problem.

File: src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java (hunk 1 above)
Comment (wottop) [reply src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java:78]: created is an OffsetDateTime. Not sure I can reduce what is here.

**Comment by wottop** [comment]
Added catch and log message for null cert.

File: src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java (hunk 1 above)
Comment (Ceiu) [reply src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java:78]: Ahh, my mistake; didn't process that this was from the DTO. Also didn't notice that we're getting millis to convert to a date. In that case, we can skip the millis bit entirely:

```
Date.from(dto.getCreated().toInstant())
```

It's too bad there's no shared interface between Instant and OffsetDateTime. The amount of `.toInstant()`s we're going to have around the code is going to be high for the foreseeable future.

**Review by Ceiu — APPROVED** [review]

**Pull request merged**